src/main/java/bio/terra/cli/service/utils/HttpUtils.java

```diff
@@ -1,12 +1,9 @@
 package bio.terra.cli.service.utils;
 
 import bio.terra.cli.exception.SystemException;
+import bio.terra.cli.utils.UserIO;
 import com.google.api.client.http.HttpStatusCodes;
-import java.io.BufferedReader;
-import java.io.DataOutputStream;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.InputStreamReader;
+import java.io.*;
```

Comment:
nit: I'm not sure why spotless/spotbugs didn't flag this, but I think the team convention is to not expand imports. I believe there's an IntelliJ setting that will keep imports expanded.

Reply:
Done. Not sure what happened; Intellij "Use single class import" was already checked for me. I'll keep an eye out for this.